core/federated/federate.c

```diff
@@ -856,6 +856,24 @@ void connect_to_federate(uint16_t remote_federate_id) {
     }
 }
 
+void perform_hmac_authentication(int rti_socket) {
```

Comment:
Method needs a comment.
It seems all it does is print the message received from the RTI?
The method name implies much more.

Reply:
Thanks for reviewing. Now this PR is ready for review. As @hokeun  commented below, it was a draft PR before.